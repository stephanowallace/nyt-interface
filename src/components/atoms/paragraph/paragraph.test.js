import React from 'react';
import { shallow } from 'enzyme';
import Paragraph from '.';

describe('Paragraph', () => {
  it('renders correctly', () => {
    const component = shallow(<Paragraph>test</Paragraph>);
    expect(component).toMatchSnapshot();
  });

  it('insert class correctly', () => {
    const component = shallow(<Paragraph className="classTest">test</Paragraph>);
    expect(component.exists('p.classTest')).toBeTruthy();
  });
});
