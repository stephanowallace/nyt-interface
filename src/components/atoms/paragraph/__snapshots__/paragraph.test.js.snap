// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph renders correctly 1`] = `
<p
  className="abstract "
>
  test
</p>
`;
